conf/distro/nilrt.conf

```diff
@@ -2,7 +2,7 @@ require nilrt.inc
 
 DISTRO_NAME = "NI Linux Real-Time"
 
-DISTRO_VERSION = "8.8"
+DISTRO_VERSION = "8.9"
 
 NILRT_FEED_NAME = "2021.0"
```

Comment:
What about this variable? Is the change coming later?

Reply:
I had been waiting to update the value of NILRT_FEED_NAME because it is used to build the feed URLs and we don't yet have ni-* feeds ready for 2021.3. However, we decided today it's better to update now and just let the feeds be broken until those feeds are created. I'll push an update to this PR for that.